src/monitorstab.py

```
@@ -0,0 +1,503 @@
+                    _ = f.write(kdl_config.print())
+                os.replace(tmp, CONFIG_PATH)
+            except Exception:
+                try:
+                    os.unlink(tmp)
+                except OSError:
+                    pass
+                raise
+
+            return True
+
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)